### PR TITLE
Feature/9 main server list

### PR DESCRIPTION
**Pull request by mihaifirst**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,11 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.5.2",
+    "@fortawesome/free-brands-svg-icons": "^6.5.2",
+    "@fortawesome/free-regular-svg-icons": "^6.5.2",
+    "@fortawesome/free-solid-svg-icons": "^6.5.2",
+    "@fortawesome/react-fontawesome": "^0.2.2",
```
Comment (vestemeansorin91) [inline package.json:17]: de ce ai nevoie de pachetele astea?

File: src/components/ServerList/ServerList.scss [hunk 2]
```diff
@@ -0,0 +1,83 @@
+.server-list {
+  min-height: 100vh;
+  width: 100px;
+  background-color: var(--grey-color-darker);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+  padding: 20px;
+  box-sizing: border-box;
```
Comment (vestemeansorin91) [inline src/components/ServerList/ServerList.scss]: https://9elements.com/css-rule-order/

cand scrii CSS-ul incearca sa te obisnuiesti cu o ordine.

File: src/components/ServerList/ServerList.scss [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      width: 100%;
+      border-radius: 50%;
+
+      &:hover {
+        border-radius: 30%;
+        transition: border-radius 0.2s ease-in;
+      }
+    }
+
+    span {
```
Comment (vestemeansorin91) [inline src/components/ServerList/ServerList.scss]: nu stiu... foarte rar gasesc situatia sa pun span / div ... mai bine pune o clasa pe el

File: src/components/ServerList/ServerList.scss (hunk 2 above)
Comment (mihaifirst) [reply src/components/ServerList/ServerList.scss]: fixed

File: src/components/ServerList/ServerList.scss [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+    &:hover {
+      border-radius: 30%;
+      transition: border-radius 0.2s ease-in;
+      background-color: var(--green-color-dark-01);
+    }
+  }
+
+  &--download-data {
+    display: flex;
```
Comment (vestemeansorin91) [inline src/components/ServerList/ServerList.scss]:
```
{
    display: flex;
    align-items: center;
    justify-content: center;
    width: 60px;
    min-height: 60px;
    border-radius: 50%;
    background-color: var(--grey-color-dark);

    &:hover {
      border-radius: 30%;
      transition: border-radius 0.2s ease-in;
      background-color: var(--green-color-dark-01);
    }
  }
```

tot codul asta il vad aplicat si la **server-list--add-server**

as redenumi clasa in 

.item (stii ca esti in contextul de **.server-list**)

iar unde ai 

 <div className="server-list--download-data">
        
      </div>

pui 2 clase, item si download-data
```
 <div className="item download-data">
        <img src={plus} fill="#000000" />
      </div>
```

File: src/index.scss [hunk 5]
```diff
@@ -15,6 +15,7 @@
   /* Colors */
   --grey-color-light: rgb(156, 152, 152);
   --grey-color-dark: rgb(50, 51, 56);
+  --grey-color-dark-02: rgb(53, 54, 60);
```
Comment (vestemeansorin91) [inline src/index.scss]: as pune mai degraba numele variabilelor sa inceapa cu, color:
ex. **--color-grey-dark-02**

pe viitor poate vrei sa pui border-radius si sa fii consistent
**--border-radius-big**

File: src/components/Register/Register.scss [hunk 6]
```diff
@@ -86,42 +86,42 @@
...
+      color: var(--color-blue-light);
     }
   }
 }
 
 .continue {
   margin-top: var(--large-size);
   border-radius: var(--short-size);
-  color: var(--grey-color-light);
+  color: var(--color-grey-light);
```
Comment (vestemeansorin91) [inline src/components/Register/Register.scss:108]: CSS order https://9elements.com/css-rule-order/

File: src/components/Register/Register.scss [hunk 7]
```diff
@@ -86,42 +86,42 @@
...
 
   input[type="checkbox"]:checked::before {
-    border-color: var(--white-color);
+    border-color: var(--color-white);
   }
 
   label {
     margin: 0;
-    color: var(--grey-color-light);
+    color: var(--color-grey-light);
```
Comment (vestemeansorin91) [inline src/components/Register/Register.scss]: CSS order https://9elements.com/css-rule-order/

File: src/components/Register/Register.scss [hunk 8]
```diff
@@ -60,16 +60,16 @@
     position: relative;
     width: var(--extra-large-size);
     height: var(--extra-large-size);
-    background-color: var(--grey-color-dark);
+    background-color: var(--color-grey-dark);
```
Comment (vestemeansorin91) [inline src/components/Register/Register.scss]: CSS order https://9elements.com/css-rule-order/

File: src/components/Register/Register.scss [hunk 9]
```diff
@@ -40,10 +40,10 @@
   justify-content: space-between;
 
   select {
-    background-color: var(--grey-color-darker);
+    background-color: var(--color-grey-darker);
```
Comment (vestemeansorin91) [inline src/components/Register/Register.scss]: CSS order https://9elements.com/css-rule-order/

File: src/components/Register/Register.scss [hunk 10]
```diff
@@ -11,11 +11,11 @@
   flex-direction: column;
 
   span {
-    color: var(--red-color);
+    color: var(--color-red);
   }
 
   > span {
-    color: var(--grey-color-light);
+    color: var(--color-grey-light);
```
Comment (vestemeansorin91) [inline src/components/Register/Register.scss]: CSS order https://9elements.com/css-rule-order/

File: src/components/Register/Register.scss [hunk 11]
```diff
@@ -1,5 +1,5 @@
 .container > span {
-  color: var(--white-color);
+  color: var(--color-white);
```
Comment (vestemeansorin91) [inline src/components/Register/Register.scss]: CSS order https://9elements.com/css-rule-order/

File: src/components/Login/Login.scss [hunk 12]
```diff
@@ -2,7 +2,7 @@
   box-sizing: border-box;
   min-width: 500px;
   padding: var(--extra-large-size);
-  background-color: var(--grey-color-dark);
+  background-color: var(--color-grey-dark);
```
Comment (vestemeansorin91) [inline src/components/Login/Login.scss:5]: CSS order https://9elements.com/css-rule-order/

File: src/components/ForgotPassword/ForgotPassword.scss [hunk 13]
```diff
@@ -2,6 +2,6 @@
   box-sizing: border-box;
   min-width: 500px;
   padding: var(--extra-large-size);
-  background-color: var(--grey-color-dark);
+  background-color: var(--color-grey-dark);
```
Comment (vestemeansorin91) [inline src/components/ForgotPassword/ForgotPassword.scss:5]: CSS order https://9elements.com/css-rule-order/

Comment (mihaifirst) [reply src/components/ForgotPassword/ForgotPassword.scss:5]: fixed

File: src/components/Login/Login.scss (hunk 12 above)
Comment (mihaifirst) [reply src/components/Login/Login.scss:5]: fixed

File: src/components/Register/Register.scss (hunk 11 above)
Comment (mihaifirst) [reply src/components/Register/Register.scss]: fixed

File: src/components/Register/Register.scss (hunk 10 above)
Comment (mihaifirst) [reply src/components/Register/Register.scss]: fixed

File: src/components/Register/Register.scss (hunk 9 above)
Comment (mihaifirst) [reply src/components/Register/Register.scss]: fixed

File: src/components/Register/Register.scss (hunk 8 above)
Comment (mihaifirst) [reply src/components/Register/Register.scss]: fixed

File: src/components/Register/Register.scss (hunk 7 above)
Comment (mihaifirst) [reply src/components/Register/Register.scss]: fixed

File: src/components/Register/Register.scss (hunk 6 above)
Comment (mihaifirst) [reply src/components/Register/Register.scss:108]: fixed

**Pull request merged**